selfdrive/test/process_replay/test_processes.py

```diff
@@ -39,6 +39,7 @@
   # Enable when port is tested and dashcamOnly is no longer set
   #("TESLA", "bb50caf5f0945ab1|2021-06-19--17-20-18--3"),      # TESLA.TESLA_AP2_MODELS
   #("VOLKSWAGEN2", "3cfdec54aa035f3f|2022-07-19--23-45-10--2"),  # VOLKSWAGEN.VOLKSWAGEN_PASSAT_NMS
+  #("VOLKSWAGEN3", "578742b26807f756/00000010--41ee3e5bec--3"),  # VOLKSWAGEN.VOLKSWAGEN_JETTA_SPORTWAGEN
```

Comment:
No need for an additional PQ process_replay route at this time. Keep the new CarTestRoute though.

```suggestion
```